nltk/parse/corenlp.py

```diff
@@ -176,15 +177,18 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         return False
 
 
-class GenericCoreNLPParser(ParserI, TokenizerI):
+class GenericCoreNLPParser(ParserI, TokenizerI, TaggerI):
     """Interface to the CoreNLP Parser."""
 
-    def __init__(self, url='http://localhost:9000', encoding='utf8'):
+    def __init__(self, url='http://localhost:9000', encoding='utf8', tagtype=None):
         import requests
 
         self.url = url
         self.encoding = encoding
 
+        assert tagtype in ['pos', 'ner', None]
```

Comment:
Throw an exception here saying that `tagtype` must be either `'pos'`, `'ner'` or `None`.